lib/tools/viewsBundler.js

```diff
@@ -0,0 +1,76 @@
+// Views Bundler: Module to expose views from server to be used also on the client
+
+module.exports = function (app) {
+  const fsr = require('./fsr')(app)
+  const fse = require('fs-extra')
+  const logger = require('./logger')(app.get('params').logging)
+  const path = require('path')
+  let fileData
+  let item
+  let fileNameAndPath
+  let fileName
+  let contents
+  let appName = app.get('appName')
+  let appDir = app.get('appDir')
+  let viewsPath = app.get('viewsPath')
+  let staticsRoot = app.get('staticsRoot')
+  let bundles = app.get('clientViewBundles')
+  let bundlesLength = Object.keys(bundles).length
+  let basePath = path.join(appDir, staticsRoot, '.build/templates')
```

Comment:
this path should be configurable

Reply:
I'm thinking of doing it similar to how the JS / CSS which is bundled up for the frontend has an output path that is specified in the configs.